testing/regress/ecl/javaembed.ecl

```diff
@@ -193,17 +193,18 @@ public class myClass {
       n = new NestedClass("nest2");
     }
   };
-  public static MyRecord transform(MyRecord in, int lim)
+  public static MyRecord testTransform(MyRecord in, int lim)
   {
     return new MyRecord(in.bfield, lim, in.dfield);
   }
 }
 ENDEMBED;
-
+/*
```

Comment:
should remove the comments

Reply:
Ok, though that will show up some more issues...